core/dbt/main.py

```diff
@@ -740,6 +740,13 @@ def parse_args(args):
         '''
     )
 
+    p.add_argument(
+        '--log-format',
+        choices=['text', 'json', None],
```

Comment:
do you think it would be clearer to use `default` instead of `None`?

Reply:
I don't think it matters much, but we can!